### ISSUE TITLE
Bug in workspace - resolve_image_exif and resolve_image_as_pil if loading an external Image

### ISSUE THREAD

**Issue by mexthecat** [OCR-D/core#838]
I got this Error Message:

```
...
  File "/var/scego/applications/docker/apps/python/venv/lib/python3.6/site-packages/ocrd/workspace.py", line 567, in image_from_page
    page_image_info = self.resolve_image_exif(page.imageFilename)
  File "/var/scego/applications/docker/apps/python/venv/lib/python3.6/site-packages/ocrd/workspace.py", line 405, in resolve_image_exif
    ocrd_exif = exif_from_filename(f.filename)
  File "/usr/lib/python3.6/tempfile.py", line 619, in __getattr__
    a = getattr(file, name)
AttributeError: '_io.BufferedRandom' object has no attribute 'filename'
```
If I try to load external Images with http for imageFilename in the page.xml

Like this:

```
...
<pc:Page imageFilename="http://link/to/image.jpg" imageWidth="2636" imageHeight="3781">
    <pc:ReadingOrder>
        <pc:OrderedGroup id="reading-order">
            <pc:RegionRefIndexed index="0" regionRef="region0000"/>
            <pc:RegionRefIndexed index="0" regionRef="region0001"/>
        </pc:OrderedGroup>
    </pc:ReadingOrder>
    <pc:TextRegion id="region0000">
...
```
I think the problem is in workspace.py on line 405
```
with download_temporary_file(image_url) as f:
    ocrd_exif = exif_from_filename(f.filename)
```

and on line 434

```
with download_temporary_file(image_url) as f:
    pil_image = Image.open(f.filename)
```

`download_temporary_file` is using `NamedTemporaryFile` - I can not see how you get filename from it.

Maybe change `f.filename` to `f.name`

**Comment by bertsky** [comment]
I fully agree – thanks for reporting @mexthecat!

Can you make that change into a PR and (install and) test it locally?

**Comment by bertsky** [comment]
@mexthecat your fix is in v2.33 and ocrd/all v2022-05-04 – can we close?

**Comment by mexthecat** [comment]
Yes - perfect - thanx
